### ISSUE TITLE
Import from reishunger.de instructions are missing

### ISSUE THREAD

**Issue by frudolph77** [hhursev/recipe-scrapers#352]
I'm heading over from this [issue](https://github.com/vabene1111/recipes/issues/438).

The ingredients and servings get imported but not the instructions, for some details see this [comment](https://github.com/vabene1111/recipes/issues/438#issuecomment-790619860)

It would be great to have a custom scraper for reishunger.de

- https://www.reishunger.de/rezepte/rezept/3035/cremiges-erdnuss-curry

Also check with this recipe they have some specials on preparing the rice in different way (pot, rice stove, microwave, ...)
- https://www.reishunger.de/rezepte/rezept/2835/suesskartoffel-kichererbsen-curry

Can you write Python and would you like to help add the scraper yourself?  We'd be glad for your assistance!  We can provide you with guidance and code review in return.  If so, tick any of the relevant boxes below:

- [ ] I'd like to try adding this scraper myself
- [ ] I'd like guidance to help me develop a scraper
- [x] I'd prefer if the `recipe-scrapers` team try to add this

If have greped the relevant part from one of their recipes, striped it down a bit and add some comments

```html
<section class="nsection recipe-preparation" readmore="800" ReadMore-60420da3674f3>
    <div class="ncontainer nbg-white" style="padding: 0 23px">
        <div>
            <h2 class="nh2">preparation</h2>
            <!-- preparation is described in the following <ol>, each single step is inside a <li> -->
            <!-- sometimes the recipes have more details on preparing the rice in the first <li>-->
            <ol class="nlist-badges" recipe-text>
                <li>
                    <p> 
                        <!-- some formatted text describing the first step -->
                        <span>Bereiten Sie</span> <b preparation="9" class="text-underline"style="display: inline !important">Basmati Reis</b> <span>zu</span>
                    </p>
                    <!-- the following div isn't always present, it contains the special instructions on preparing the rice -->
                    <!-- if the div missing then the previous <p> has no <span> -->
                    <!-- for a simplified version of the scraper these special instructions can be omitted and only use the previous <p> as step one-->
                    <div preparation="9" style="max-width: 100%; width: 100%">
                        <!-- the content of the following div is not relevant -->
                        <div style="display: flex">
                            <div pmethod="9-pot" style="display: flex; flex-direction: column; align-items: center; padding: 5px 10px; max-width: 25%;" class="method-select">
                                <img style="max-width: 50px; width: 100%" src="https://www.reishunger.de/assets/img/icons/pot.svg?7a6dfdba9ca1dba102ce03ddfe7d0915"/>
                                <span>pot</span> <i class="fa fa-caret-down"></i>
                            </div>
                            <div pmethod="9-electrical" style=" display: flex; flex-direction: column; align-items: center; padding: 5px 10px; max-width: 25%;" class="method-select">
                                <img style="max-width: 50px; width: 100%" src="https://www.reishunger.de/assets/img/icons/electrical.svg?62f0cdb4b6cb89d467a2f869b6a55364" />
                                <span>rice cooker</span>
                                <i class="fa fa-caret-down"></i>
                            </div>
                            <!-- ... -->
                        </div>
                        <!-- interesting parts follow here -->
                        <div pmethod="9-pot" style="padding: 15px; background: #eee">
                            <div style=" display: flex; align-items: center; justify-content: space-between;">
                                <p class="nh3" style="font-size: 21px">prep in pot</p>
                                <button class="nbtn nbtn-grey-light"> <i class="fa fa-times"></i> </button>
                            </div>
                            <ul class="list-alpha">
                                <li style="line-height: 1">pot step 1</li>
                                <li style="line-height: 1">pot step 2</li>
                                <!-- ^ <li> repeated multiple times-->
                            </ul>
                        </div>
                        <div pmethod="9-electrical" style="padding: 15px; background: #eee">
                            <div style="display: flex; align-items: center; justify-content: space-between;">
                                <p class="nh3" style="font-size: 21px">prep in rice stove</p>
                                <button class="nbtn nbtn-grey-light"> <i class="fa fa-times"></i> </button>
                            </div>
                            <ul class="list-alpha">
                                <li style="line-height: 1">rice stove step 1</li>
                                <li style="line-height: 1">rice stove step 2</li>
                                <!-- ^ <li> repeated multiple times-->
                            </ul>
                        </div>
                        <!-- ^ <div> repeated multiple times -->
                    </div>
                </li>
                <li><p style="line-height: 1">step 2</p></li>
                <li><p style="line-height: 1">step 3</p></li>
                <!-- ^ <li> repeated multiple times -->
            </ol>
        </div>
    </div>
</section>
```
---
EDIT: TODO list

- [ ] Base scraper for single instruction
- [ ] Multi instruction(?)

**Comment by frudolph77** [comment]
Thanks for this really quick enhancement.

**Comment by vabene1111** [comment]
ok it appears that reishunger uses two different formats depending on if they offer different cooking methods or not. 

I found a way to parse the multi method recipes and will integrate it, PR will follow soon. There is one decision to be made and that is if the instructions should include the instructions for all 4 cooking methods or if only the normal cooking pot should be returned. 

I personally prefer to return all options in one single text block so you do not loose any information depending on which version you want.

**Comment by jayaddison** [comment]
Fixes addressing this have been published in version 14.17.0 - please give that a try and let us know if any problems remain.  Thanks!
